### PR TITLE
@orta => [Martsy] Fix crash due to NSTimer based retain cycle.

### LINKED ISSUE #557: Fix crash in - [ARInternalMobileWebViewController webViewDidFinishLoad:], line 169
### Stacktrace ###

<pre>ARInternalMobileWebViewController;webViewDidFinishLoad:;ARInternalMobileWebViewController.m;169</pre>

### Reason ###

objc_msgSend() selector name: setContentLoadStateTimer:

### Link to HockeyApp ###

* [https://rink.hockeyapp.net/manage/apps/37029/crash_reasons/38397576](https://rink.hockeyapp.net/manage/apps/37029/crash_reasons/38397576)

### PR DESCRIPTION
**Pull request by alloy**
If a user would navigate back before the webview fully finished loading,
the VC would be retained by the timer and released once the timer was
invalidated, which in turn would lead to a crash if the VC was
referenced after invalidating the timer.

Fixes #557. (See there for more details.)

Tested on :iphone:

### CODE REVIEW

**Comment by orta** [comment]
( will need a rebase )

**Comment by alloy** [comment]
Yeah, clearly it does now :smiling_imp:

File: Podfile [hunk 1]
```diff
@@ -54,6 +64,7 @@ target 'Artsy' do
   pod 'ARCollectionViewMasonryLayout', '~> 2.1', '>= 2.1.2'
   pod 'ARGenericTableViewController', '1.0.2'
   pod 'FLKAutoLayout', '0.1.1'
+  pod 'TPDWeakProxy', '1.1.0'
```
Comment (orta) [inline Podfile]: dandy you just ask me to remove these kind of direct versioning?

Comment (alloy) [inline Podfile]: I did, exactly because I had to do it again to conform to the rest of the file’s style :)

**Comment by alloy** [comment]
Rebased!

File: Podfile (hunk 1 above)
Comment (orta) [inline Podfile]: hah, not one to make :wave:s - I understand

**Comment by alloy** [comment]
Bump.

**Comment by alloy** [comment]
Merging it!

**Pull request merged**